custom_model_runner/datarobot_drum/drum/language_predictors/base_language_predictor.py

```diff
@@ -134,28 +134,28 @@ def _init_mlops(self):
         if self._params.get("model_id", None):
             self._mlops.set_model_id(self._params["model_id"])
 
+        self._configure_mlops()
+
         if self.supports_chat():
-            self._configure_mlops_for_chat()
-        else:
-            self._configure_mlops_for_non_chat()
```

Comment:
Seems like we can delete the unused method now:
https://github.com/datarobot/datarobot-user-models/blob/984e1937c008c2ee3c4fd17f68c2388c5088d79c/custom_model_runner/datarobot_drum/drum/language_predictors/base_language_predictor.py#L179-L180